examples/request_api.py

```diff
@@ -47,14 +44,14 @@ def run_proc2():
 
 
 async def proc2_worker():
-    config = ConnectionConfig.from_name("e2")
-    async with AsyncioEndpoint.serve(config) as endpoint:
-        await endpoint.connect_to_endpoints(ConnectionConfig.from_name("e1"))
```

Comment:
The pattern of having an endpoint connect to itself is no longer valid under this new paradigm.  cc @cburgdorf

Reply:
But we are still able to subscribe to events that originate from our very own endpoint, right? Because I think that is pretty essential.

Reply:
Yes, I'm going to fix this.